p2p/server.go

```diff
@@ -873,9 +876,13 @@ func (srv *Server) listenLoop() {
 			continue
 		}
 		if remoteIP != nil {
-			fd = newMeteredConn(fd, true, remoteIP)
+			var addr *net.TCPAddr
```

Comment:
@fjl Could you please take a look? I didn't know what is the proper solution of this merge conflict.

Reply:
The relevant information is that I use the `TCPAddr` instead of the `IP`.